server/api/card.js

```diff
@@ -70,9 +70,6 @@ router.post('/make25/forRoom/:roomId', async (req, res, next) => {
     const { roomId } = req.params;
     const { selectedWordPackId } = req.body;
 
-    // Create a new board to put the 25 cards into
-    const board = await Board.create();
```

Comment:
i moved this further down to be next to the room/board association for easier readability